corehq/apps/es/management/commands/restore_es_snapshot.py

```diff
@@ -0,0 +1,78 @@
+from django.core.management.base import BaseCommand, CommandError
+from datetime import date, timedelta
+from corehq.elastic import get_es_new
+from elasticsearch.client import SnapshotClient, IndicesClient
+from django.conf import settings
+from pillowtop.utils import get_all_pillow_instances
+from corehq.apps.hqadmin.models import ESRestorePillowCheckpoints
+from pillowtop.checkpoints.manager import DEFAULT_EMPTY_CHECKPOINT_SEQUENCE
+
+
+class Command(BaseCommand):
+    help = ("Restores full ES cluster or specific index from snapshot. "
+            "Index arguments are optional and it will default to a full "
+            "cluster restore if none are specified")
+    args = "days_ago <index_1> <index_2> ..."
+
+    def handle(self, *args, **options):
+        print "Restoring ES indices from snapshot"
+        if len(args) < 1:
+            raise CommandError('Usage is restore_es_snapshot %s' % self.args)
+        date = self.get_date(args)
+        indices = self.get_indices(args)
+        es = get_es_new()
+        client = self.get_client_and_close_indices(es, indices)
```

Comment:
Should add in a user confirmation here before going ahead, especially since the default is to close ALL indices.